### DISCUSSION TITLE
Is my computer powerful enough to be useful?

### DISCUSSION THREAD

**Discussion by clothstir** [lichess-org/fishnet#232] [Q&A]
I have an unused Macbook (2 GHz Quad-Core Intel Core i5 ; 16 GB 3733 MHz LPDDR4X) is this powerful enough to usefully run fishnet (and nothing else)?

I've installed fishnet and tried running it, but I just repetitively get this message in verbose mode:

```
D: Going idle for 30s.
D: User wait: 30s due to 30s for oldest 0ns, system wait: 3600s due to 3600s for oldest 0ns
D: Going idle for 30s.
D: User wait: 30s due to 30s for oldest 0ns, system wait: 3600s due to 3600s for oldest 0ns
D: Going idle for 30s.
```

My fishnet.ini

```
[fishnet]
cores=8
userbacklog=short
key=REDACTED
systembacklog=long
```

Thanks in advance!

**Comment by niklasf** [comment]
Hi, thanks for running fishnet.

The configuration

```
userbacklog=short
systembacklog=long
```

means your client is configured to help only in "emergencies", when a backlog of analysis requests is building up. You can rerun `./fishnet configure` or manually set both to `0` to contribute more frequently. Either way, you're helping.
